src/Streamly/Internal/LZ4.hs

```diff
@@ -13,6 +13,12 @@ module Streamly.Internal.LZ4
     ( debugD
     , debug
 
+    -- * Configuration
+    , Config
```

Comment:
Do not export `Config`. We are exporting the setter functions, this type would be opaque.

Reply:
I'm only exporting the type.